.cirrus.yml

```diff
@@ -123,6 +126,91 @@ task:
       pg-ci-${CIRRUS_TASK_NAME}-${DATE}
 
 
+task:
+  matrix:
+    - name: openbsd-7-0-vanilla
+      env:
+        PACKERFILE: packer/bsd/openbsd_vanilla.pkr.hcl
+
+    - name: netbsd-9-2-vanilla
```

Comment:
Could these be part of the existing packer invocation? Then we'd not have to repeat all the boilerplate again?

Reply:
There are small differences like;

Existing packer: 
```    
packer build \
      -only "*.${CIRRUS_TASK_NAME}" \
      -timestamp-ui \
      -force \
      -var "image_date=$DATE" \
      -var gcp_project=$GCP_PROJECT \
      "$PACKERFILE"
```
OpenBSD / NetBSD:
```
    packer build \
      -timestamp-ui \
      -force \
      -var "image_date=$DATE" \
      -var "image_name=${CIRRUS_TASK_NAME}" \
      -var "bucket=$BUCKET" \
      -var gcp_project=$GCP_PROJECT \
      "$PACKERFILE"
```

Existing packer does not include
```
      -var "image_name=${CIRRUS_TASK_NAME}" \
      -var "bucket=$BUCKET" \
```
so, we need to add these variables to existing packer files, otherwise packer will complain about that. Also, their skip cases are different.